.github/workflows/pre-commit.yml

```diff
@@ -6,13 +6,20 @@ jobs:
   prettier:
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout
+      - name: Checkout Repository
         uses: actions/checkout@v3
 
-      - name: Setup python
-        uses: actions/setup-python@v2
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v4
+        with:
```

Comment:
Python was running into an issue on my PR for some reason.  I update the set up and precommit action seemed to fix it.